styles/styles.css

```diff
@@ -26,6 +26,14 @@
     font-weight: var(--font-weight-bold);
 }
 
+@font-face {
+    font-family: 'Open Sans';
+    src: url("/fonts/Open_Sans_700.woff2") format("woff2");
+    font-display: swap;
+    font-style: normal;
+    font-weight: var(--font-weight-heavy);
+}
+
```

Comment:
@solaris007 move custom font loading to lazy-styles and use a fallback based on a system font to remove them from the main thread and prevent delaying FCP and TBT. See [here](https://www.hlx.live/developer/font-fallback) for nice helper tool to create fallback font definitions.

Reply:
@rofe - thank you, i have added:
https://github.com/hlxsites/choice/issues/23 and
https://github.com/hlxsites/elco-aerotop-sg/issues/138